### PR TITLE
fix: faq

### PR DESCRIPTION
**Pull request by vinckr**
The FAQ currently is outdated, see questions on SPAs and others. 
This patch 
- moves the FAQ to the Debug & Help section
- renames it to Tips & Troubleshooting
- moves many of the questions to documents where they fit better, reformatted and with added information where needed.
- also some other spelling/format fixes

PR in docusaurus-template to remove the FAQ there: 
https://github.com/ory/docusaurus-template/pull/87

## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md).
- [x] I have referenced an issue containing the design document if my change
      introduces a new feature.
- [x] I am following the
      [contributing code guidelines](../blob/master/CONTRIBUTING.md#contributing-code).
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security. vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [ ] I have added tests that prove my fix is effective or that my feature
      works.
- [x] I have added or changed [the documentation](docs/docs).

### CODE REVIEW

**Review by aeneasr — APPROVED** [review]

**Comment by aeneasr** [comment]
Thank you, this looks great! The CI is failing because some files are formatted incorrectly. To format them, run:

```
$ make format
$ git commit -a -m "styles: format code"
$ git push
```

Thank you!

**Review by zepatrik — COMMENTED** [review]
forgot to click on submit... maybe you can open a follow up PR?

**Comment by vinckr** [comment]
Thanks a lot for the review @zepatrik, I will open a follow up PR.

File: docs/docs/guides/sign-in-with-github-google-facebook-linkedin.mdx [hunk 1]
```diff
@@ -1147,3 +1145,66 @@ Spotify is now an option to log in via Kratos.
...
+following issue as well:
+[Add random function to the jsonnet library](https://github.com/ory/kratos/issues/988).
+
+## How to skip Social Login Link-Up
+
+If multiple
+[registration methods](../docs/self-service/flows/user-registration.mdx) are
+configured, users may be redirected to a social login provider
+to link the accounts after the email verification. To skip this link-up add the
+session hook after the registration method (e.g. `password`):
```
[the remark this answers is not in the document]
Comment (vinckr) [reply docs/docs/guides/sign-in-with-github-google-facebook-linkedin.mdx]: The session hook will not skip the link up? 
At least for the user who asked this, it solved the issue. 
Should I revert this and test it before we add it? 
It also seems like an edgecase...

**Pull request merged**